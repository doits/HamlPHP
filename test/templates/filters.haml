!!! 5
%html
  %head
    :css
      body { color: #000; }
      
      p {
        color: #fff;
      }
      
      #container { color: #fff; }
    :plain
      %p should not render
      / neither this
    
    :javascript
      $(function() {
        function helloWorld() {
          alert('Hello world');
        }
      });
    :php
      $variable = 1;
      
      function helloWorld() {
        return 'Hello world';
      }
  %body
    %p test
    %p
      :plain
        %p testing plain
          nesting
            nesting
            = 2 + 2