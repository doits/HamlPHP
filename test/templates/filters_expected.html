<!DOCTYPE html>
<html>
  <head>
    <style type="text/css">
      body { color: #000; }
      p {
        color: #fff;
      }
      #container { color: #fff; }
    </style>
    %p should not render
    / neither this
    <script type="text/javascript">
      $(function() {
        function helloWorld() {
          alert('Hello world');
        }
      });
    </script>
    <?php
    $variable = 1;
    function helloWorld() {
      return 'Hello world';
    }
    ?>
  </head>
  <body>
    <p>test</p>
    <p>
      %p testing plain
        nesting
          nesting
          = 2 + 2
    </p>
  </body>
</html>
